Modules/OrgUnit/interfaces/class.ilOrgUnitPositionAccessHandler.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * Class ilOrgUnitPositionAccessHandler
+ *
+ * @author Fabian Schmid <fs@studer-raimann.ch>
+ */
+interface ilOrgUnitPositionAccessHandler {
+
+	const PERMISSION_VIEW_LEARNING_PROGRESS = 'viewlp';
+	const PERMISSION_VIEW_TEST_RESULTS = 'viewtstr';
```

Comment:
If I understand correctly, this should only be a temporary solution. We would need to enumerate every possible permission for positions for every object here, right? This IMO is bad for modularity. I currently also cannot see how this scales to plugin objects. I think that a "correct" solution would also entail the permissions in the AccessHandler in general, so we might want to tackle this for the next release.